schema/filter_artworks.js

```diff
@@ -145,6 +145,10 @@ function filterArtworks(primaryKey) {
         delete gravityOptions.medium;
       }
 
+      if (accessToken) {
+        return gravity.with(accessToken)('filter/artworks', gravityOptions);
+      }
+
       return gravity('filter/artworks', gravityOptions);
```

Comment:
Oh, ok missed that. Let me check, I think it should be fine because `gravity.with(null)` just would get the access token stripped out.

But, I'll also check if it bypasses caching or not. We definitely wouldn't want to bypass caching with a null token.